.azure-pipelines/all.yml

```diff
@@ -9,7 +9,7 @@ pr:
     - master
   paths:
     include:
-    - datadog_checks_base/datadog_checks/*
+    - datadog_checks_base/*
```

Comment:
Why?

Reply:
Commented here: https://github.com/DataDog/integrations-core/pull/4711#discussion_r332880839

Reply:
That doesn't explain _why_ though

Reply:
Modifying its tox.ini would not impact any other checks